pkg/core/contracts/contracts.go

```diff
@@ -289,3 +294,98 @@ func (ac *AudiusContracts) GetServiceTypeManagerContract() (*gen.ServiceTypeMana
 	ac.ServiceTypeManager = contract
 	return contract, nil
 }
+
+/* Utilities */
+
+type Node struct {
+	Owner               geth.Address
+	Endpoint            string
+	BlockNumber         *big.Int
+	DelegateOwnerWallet geth.Address
+}
```

Comment:
Note that we have a similar struct in registrar/audius_api_gateway.go. We don't have to reuse it this time, but just fyi

Reply:
ah thanks, would be nice to merge these at some point